templates/web/base/admin/triage/_list-filters.html

```diff
@@ -0,0 +1,53 @@
+[% select_category = BLOCK %]
```

Comment:
I think I factored out the state dropdown in the current Zurich work, in which case this and the base reports/_list-filters.html template are basically the same and could be factored together.

Reply:
Yup, not quite sure how best to use that in this branch at the moment though given the Zurich things aren't merged yet.

Reply:
Yeah, thought that'd be in by now. Anyway, can be left to after, no biggie.